Worker.Extension.Sql/README.md

```diff
@@ -0,0 +1,180 @@
+# SQL Extension .NET Worker
+
+Welcome to the Sql Extension .NET Worker Repository. The .NET Worker provides .NET 6 support for SQL Bindings in Azure Functions, introducing an **Isolated Model**, running as an out-of-process language worker that is separate from the Azure Functions runtime. This allows you to have full control over your application's dependencies as well as other new features like a middleware pipeline. A .NET Isolated function app works differently than a .NET Core 3.1 function app. For .NET Isolated, you build an executable that imports the .NET Isolated language worker as a NuGet package. Your app includes a [`Program.cs`](vscode-file://vscode-app/c:/Users/makoripa/AppData/Local/Programs/Azure%20Data%20Studio/resources/app/out/vs/code/electron-browser/workbench/samples/samples-outofproc/Program.cs) that starts the worker.
+
+## Binding Model
+
+.NET Isolated introduces a new binding model, slightly different from the binding model exposed in .NET Core 3 Azure Functions. More information can be [found here](https://github.com/Azure/azure-functions-dotnet-worker/wiki/.NET-Worker-bindings). Please review our samples for usage information.
+
+## Samples
+
+You can find samples on how to use Sql Extension .NET Worker under `samples\samples-outofproc` ([link](https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-outofproc)).
+
+## Create and run .NET Isolated functions
+
+**Note: Visual Studio and Visual Studio Code support is on the way. In the meantime, please use `azure-functions-core-tools` or the sample projects as a starting point.**  
+
+### Install .NET 6.0
+
+Download .NET 6.0 [from here](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+### Install the Azure Functions Core Tools
+
+To download Core Tools, please check out our docs at [Azure Functions Core Tools](https://github.com/Azure/azure-functions-core-tools)
+
+### Create a .NET Isolated project
+
+In an empty directory, run `func init` and select `dotnet (Isolated Process)`.
+
+### Add a function
+
+Run `func new` and select `HttpTrigger` trigger. Fill in the function name.
+
+### Configure function App
+
+1. Get your SQL connection string
+    
+    Local SQL Server - Use this connection string, replacing the placeholder values for the database and password.  
+      
+    `Server=localhost;Initial Catalog={db_name};Persist Security Info=False;User ID=sa;Password={your_password};` Azure SQL Server - Browse to the SQL Database resource in the \[Azure portal\](https://ms.portal.azure.com/)  
```

Comment:
Fix the portal link - it looks like this currently since you're escaping the brackets

![image](https://user-images.githubusercontent.com/28519865/200739319-93a1cbe5-15c8-4673-8b17-091eec8805a5.png)